shell.py

```diff
@@ -245,12 +251,19 @@ class NextTurn(Command):
 
     keywords = ['next']
 
+    @manager.registry.add_binding(Keys.ControlN)
     def do_command(self, *args):
-        if not self.game.tm:
+        if not game.tm:
+            if not hasattr(self, "game"):
+            # this feels very hacky however it looks like key binding
+            # takes over object. To me it looks like we might want to change
+            # approach on how we load commands.
```

Comment:
Agreed, this is super weird. Also I was never in love with the command registration setup anyway, so I am definitely open to fixing that up at some point!

Reply:
sure thing.